spec/masamune/actions/execute_spec.rb

```diff
@@ -25,6 +25,14 @@
     Class.new do
       include Masamune::HasEnvironment
       include Masamune::Actions::Execute
+
+      def before_execute
+        fail
```

Comment:
when `Hive` is wrapped by `AwsEmr`, (`after_execute`)[https://github.com/socialcast/masamune/blob/master/lib/masamune/commands/hive.rb#L109] inadvertently called by (`execute`)[https://github.com/socialcast/masamune/blob/master/lib/masamune/commands/aws_emr.rb#L80]. This resulted in (`@buffer` being closed during `handle_stdout`)[https://github.com/socialcast/masamune/blob/master/lib/masamune/commands/hive.rb#L127]